netlify.toml

```diff
@@ -0,0 +1,10 @@
+[build]
+  base = "web"
+  publish = "dist"
+
+  command = "yarn run build"
+
+    [build.environment]
```

Comment:
I am curious: Is such indentation required or some convention I haven't encountered yet?

Reply:
In TOML, indentation is optional. You can see this in my other comment, but if a section is a subsection of another, I tend to indent it. Otherwise, I don't indent.

Reply:
@yarikoptic some of my indentation was accidental, so don't read too much into it :-)